python/lsst/meas/extensions/shapeHSM/_hsm_higher_moments.py

```diff
@@ -369,9 +369,13 @@ def measure(self, record, exposure):
         except KeyError:
             raise measBase.FatalAlgorithmError("'ext_shapeHSM_HsmPsfMoments' plugin must be enabled.")
 
+        centroid = self.centroidExtractor(record, self.flagHandler)
+
+        if record["ext_shapeHSM_HsmPsfMoments_flag"]:
```

Comment:
Would an in-line comment above this `if` help explain the rationale for this new addition?